### ISSUE TITLE
Missing template : "home/comments"

### ISSUE THREAD

**Issue by jiteshjha** [publiclab/plots2#637]
### What happened just before the problem occurred
The comments sub-section for any profile page could not be loaded because of the change #525. 
Error(console) : Missing template: home/comments partial

### Relevant URLs

https://publiclab.org/profile/warren#comments

Screenshot : 
![screenshot from 2016-07-27 21 58 47](https://cloud.githubusercontent.com/assets/10262106/17183506/6d1bbb1c-5445-11e6-906f-af804dd71884.png)

**Comment by jiteshjha** [comment]
https://github.com/publiclab/plots2/blob/master/app/controllers/users_controller.rb#L205 should be changed.

**Comment by jywarren** [comment]
ah, good catch! Thanks, do you think you could fix and submit a test for
this as well?

On Wed, Jul 27, 2016 at 12:33 PM, Jitesh Kumar Jha <notifications@github.com

**Comment by ananyo2012** [comment]
@jywarren Can you wait on it for a bit. I have addressed this issue in #628. See I have mentioned it here: https://github.com/publiclab/plots2/pull/628#discussion_r71965866

**Comment by jywarren** [comment]
Oh, super. Thanks Ananyo. Sorry I missed that and thanks for catching it
here. So many PRs and contributions, which is great, but sometimes hard to
follow!

**Comment by ananyo2012** [comment]
Yeah it is in fact hard to follow with so may PRs. I am trying to address any view related issue in the Q & A system in #628

**Comment by ananyo2012** [comment]
Closing this solved in #628
